ios/react-native-navigation-sdk/NavAutoModule.m

```diff
@@ -374,6 +380,21 @@ + (void)unregisterNavAutoModuleReadyCallback {
   });
 }
 
+RCT_EXPORT_METHOD(isAutoScreenAvailable
+                  : (RCTPromiseResolveBlock)resolve rejecter
+                  : (RCTPromiseRejectBlock)reject) {
+  dispatch_async(dispatch_get_main_queue(), ^{
+    BOOL hasViewController = self->_viewController != nil;
+    resolve([NSNumber numberWithBool:hasViewController]);
+  });
+}
+
+- (void)onScreenStateChange:(BOOL)available {
+  NSMutableDictionary *map = [NSMutableDictionary dictionary];
```

Comment:
It doesn't seem this map instance is used within its scope?

Reply:
true, please remove it

Reply:
Fixed